test/fixtures/profiles.yml

```diff
@@ -1,9 +1,2 @@
-# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
-
-one:
-  nickname: MyString
-  profile_picture: MyString
-
-two:
-  nickname: MyString
-  profile_picture: MyString
+abdullah:
+  user: abdullah
```

Comment:
I love fixtures. In my experience, they work better when they are an approximation of real world data.

Reply:
Make sense